Sources/PaymentsCore/Networking/HTTP.swift

```diff
@@ -12,12 +12,12 @@ class HTTP {
         self.coreConfig = coreConfig
     }
     
-    func performRequest<T: APIRequest>(endpoint: T) async throws -> (T.ResponseType) {
-        guard let request = endpoint.toURLRequest(environment: coreConfig.environment) else {
+    func performRequest<T: APIRequest>(_ request: T) async throws -> (T.ResponseType) {
```

Comment:
Take it or leave it, but wondering if this would be more readable as:
```
let (_) = try await http.performRequest(with: analyticsEventRequest)
```

This is similar to what we do with the urlSession on line 20 below.

Reply:
I'm a little confused, which line are you referring to changing?

Reply:
Ah, yeah GitHub's highlighting is _less than ideal sometimes_. I was referencing this line:
```suggestion
    func performRequest<T: APIRequest>(with request: T) async throws -> (T.ResponseType) {
```